test/Hydration-test.js

```diff
@@ -9,426 +9,25 @@ describe('Hydration', () => {
-        "date": "2019/09/18",
-        "numOunces": 98
-      },
-      {
-        "userID": 35,
-        "date": "2019/09/18",
-        "numOunces": 58
-      },
-      {
-        "userID": 36,
-        "date": "2019/09/18",
-        "numOunces": 94
-      },
-      {
-        "userID": 37,
-        "date": "2019/09/18",
-        "numOunces": 34
-      },
-      {
-        "userID": 38,
-        "date": "2019/09/18",
-        "numOunces": 66
-      },
-      {
-        "userID": 39,
-        "date": "2019/09/18",
-        "numOunces": 86
-      },
-      {
-        "userID": 40,
-        "date": "2019/09/18",
-        "numOunces": 77
-      },
-      {
-        "userID": 41,
-        "date": "2019/09/18",
-        "numOunces": 74
-      },
-      {
-        "userID": 42,
-        "date": "2019/09/18",
-        "numOunces": 51
-      },
-      {
-        "userID": 43,
-        "date": "2019/09/18",
-        "numOunces": 84
-      },
-      {
-        "userID": 44,
-        "date": "2019/09/18",
-        "numOunces": 83
-      },
-      {
-        "userID": 45,
-        "date": "2019/09/18",
-        "numOunces": 85
-      },
-      {
-        "userID": 46,
-        "date": "2019/09/18",
-        "numOunces": 66
-      },
-      {
-        "userID": 47,
-        "date": "2019/09/18",
-        "numOunces": 57
-      },
-      {
-        "userID": 48,
-        "date": "2019/09/18",
-        "numOunces": 21
-      },
-      {
-        "userID": 49,
-        "date": "2019/09/18",
-        "numOunces": 63
-      },
-      {
-        "userID": 50,
-        "date": "2019/09/18",
+        "date": "2019/06/15",
         "numOunces": 69
-      },
-      {
-        "userID": 1,
-        "date": "2019/09/19",
-        "numOunces": 53
-      },
-      {
-        "userID": 2,
-        "date": "2019/09/19",
-        "numOunces": 27
-      },
-      {
-        "userID": 3,
-        "date": "2019/09/19",
-        "numOunces": 77
-      },
-      {
-        "userID": 4,
-        "date": "2019/09/19",
-        "numOunces": 73
-      },
-      {
-        "userID": 5,
-        "date": "2019/09/19",
-        "numOunces": 73
-      },
-      {
-        "userID": 6,
-        "date": "2019/09/19",
-        "numOunces": 93
-      },
-      {
-        "userID": 7,
-        "date": "2019/09/19",
-        "numOunces": 47
-      },
-      {
-        "userID": 8,
-        "date": "2019/09/19",
-        "numOunces": 71
-      },
-      {
-        "userID": 9,
-        "date": "2019/09/19",
-        "numOunces": 99
-      },
-      {
-        "userID": 10,
-        "date": "2019/09/19",
-        "numOunces": 52
-      },
-      {
-        "userID": 11,
-        "date": "2019/09/19",
-        "numOunces": 65
-      },
-      {
-        "userID": 12,
-        "date": "2019/09/19",
-        "numOunces": 32
-      },
-      {
-        "userID": 13,
-        "date": "2019/09/19",
-        "numOunces": 62
       }
     ];
+    oneData = new Hydration(hydrationData[0]);
```

Comment:
Since this data is now going to be one object, we don't need to keep it in an array, too. `hydrationData` can just be one object, not an object in an array.